src/JobPanel/backend/service_base.py

```diff
@@ -214,24 +218,61 @@ def _process_requests(self):
                 data = request['data']
             assert( 'action' in request.keys() )
             answer = self.call_action(request['action'], data)
```

Comment:
Pokud to budes delat takto, tak to bude potreba. Mozna by se implementace zjednodusila, pokud by `call_action` nevracelo answer, ale vlakno a answer dist by to naopak prebralo jako parametr a naplnilo.